### ISSUE TITLE
[jsx-pascal-case] label variable fail

### ISSUE THREAD

**Issue by yordis** [yannickcr/eslint-plugin-react#343]
The label tag is failing on this rule

Imported JSX component label must be in PascalCase  `react/jsx-pascal-case`

**Comment by yannickcr** [comment]
Can you give me a code sample that trigger that invalid lint errors ?

**Comment by playpauseandstop** [comment]
I ran into same problem today with this code,

```js
  render() {
    const {label} = this.props;
    return <label className="...">{label}</label>;
  }
```

From my perspective it happens, cause ESLint seeing defined `label` var in current scope and considers warning for pascal-case in `<label>` JSX tag.

ps. I fixed the error with `const {label: labelText} = this.props;`

**Comment by yordis** [comment]
@yannickcr yep the same here
`<label htmlFor={name}>{label}</label>`

**Comment by yannickcr** [comment]
Seems to be a duplicate of #329
